src/components/Terminal/Terminal.style.js

```diff
@@ -80,16 +79,16 @@ export const TerminalWrapper = styled.div`
     width: 0;
     height: 0;
   }
-
+  
   .code-wrapper {
-     pre {
+    pre {
       padding: 0;
       margin: 0;
       width: 100%;
       height: 100%;
       font-style: normal;
       font-weight: normal;
-      font-size: 12px;
+      font-size: 10px;
```

Comment:
The other two changes are an improvement and help manage text overflow. This size change, though... could you revert this?

Reply:
Okay, I'll revert it.